README.md

```diff
@@ -1,5 +1,225 @@
 # Uniara Virtual API
 
+API to make integration with Uniara Virtual system.
+
+## Using (documentation)
+To use the api, follow the next steps.
```

Comment:
I don't think you need the comma here :smile: and I think API looks better in caps..